TauAnalysis/MCEmbeddingTools/test/runtests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -ex
+
+##____________________________________________________________________________||
+function die { echo $1: status $2 ; exit $2; }
+
+
+
+
+# echo '{
+#"274199" : [[1, 180]]
+#}' > step1_lumiRanges.log  2>&1
+ 
+# (das_client --limit 0 --query 'file dataset=/DoubleMuon/Run2016B-v2/RAW run=274199') | sort -u > step1_dasquery.log  2>&1
+#due to frequent failures of das_client: use file directly
+
+#echo "/store/data/Run2016B/DoubleMuon/RAW/v2/000/274/199/00000/02893BCB-6426-E611-B266-02163E012552.root" > step1_dasquery.log
+
+#cmsDriver.py selecting -s RAW2DIGI,L1Reco,RECO,PAT --runUnscheduled --data --scenario pp --conditions auto:run2_data --era Run2_2016_HIPM --runUnscheduled --eventcontent RAWRECO --datatier RAWRECO --customise Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2016,TauAnalysis/MCEmbeddingTools/customisers.customiseSelecting --filein filelist:step1_dasquery.log --lumiToProcess step1_lumiRanges.log --fileout step2.root -n 200 --nThreads 32  
+
+
+## This is a PRE SKIMED dataset, so  
+echo "/store/user/swayand/Emmbeddingfiles/Emmbedding_testInput3.root"  > file_list.txt
```

Comment:
once the buildfile name is corrected, let's see if automated jenkins tests can read this file.
.. most likely it will